package.json

```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0",
   "description": "Mozfest site",
   "author": "Mozilla Foundation",
+  "repository": "mozilla/mozillafestival.org",
+  "license": "MPL-2.0",
```

Comment:
Just realized we don't have a `LICENSE` file on this repo. Do you mind adding it in this PR @alanmoo ?